rpm/sailfish-setup.spec

```diff
@@ -33,33 +33,27 @@ if ! getent passwd sailfish-actdead >/dev/null ; then
     useradd -r -g sailfish-actdead -d / -s /sbin/nologin sailfish-actdead || :
 fi
 
-# TODO : This should be moved to the first boot (JB#48049)
-# Make sure that device owner exists.
-groupadd -fg 100000 nemo || :
-if ! getent passwd nemo >/dev/null ; then
-    # Requirements - source component in brackets:
-    # video for display (setup), input input devices (systemd), audio for itself (setup)
-    # users (setup)
-    useradd -g nemo -G "video,input,audio,users" -u 100000 -m nemo || :
-fi
-
 groupadd -rf oneshot || :
-usermod -a -G oneshot nemo || :
 
 groupadd -rf sailfish-system || :
-usermod -a -G sailfish-system nemo || :
 usermod -a -G sailfish-system sailfish-mdm || :
```

Comment:
While at it, could you add this usermod to the list of Requires(pre) commands, please? Others seem to be there already.

Reply:
Done